src/main/java/sirius/biz/protocol/AuditLogController.java

```diff
@@ -31,13 +32,19 @@
      */
     public static final String PERMISSION_AUDIT_LOGS = "permission-audit-logs";
 
+    /**
+     * Names the permissions required to view the audit log page.
+     */
+    private static final String PERMISSION_VIEW_AUDIT_LOGS = "permission-view-audit-log";
+
```

Comment:
whats the difference to the permission above? can you expand the javadoc so this is clear?

Reply:
I tried to clear it up so the permissions are less confusing. `permission-audit-logs` allows the user to see not only his own audit log, but all audit log entries of the user's tenant. `permission-view-audit-logs` allows the user to view the audit-log page at all.